themes/bootstrap3/js/lightbox.js

```diff
@@ -523,6 +523,13 @@ VuFind.register('lightbox', function Lightbox() {
     });
     _modal.on("shown.bs.modal", function lightboxShown() {
       bindFocus();
+
+      // Disable bootstrap-accessibility.js "enforceFocus" events.
+      // retainFocus() above handles it better.
+      // This is moot once that library (and bootstrap3) are retired.
+      var focEls = _modal.find(":tabbable");
+      var lastEl = $(focEls[focEls.length - 1]);
+      $(lastEl).off('keydown.bs.modal');
```

Comment:
Well sussed! I notice that it applied to the first element as well. Might as well take care of both, right?

https://github.com/paypal/bootstrap-accessibility-plugin/blob/main/src/js/modal.js#L16

Reply:
Sure, I made that change, I don't think there is any impact but I agree it's better to remove both listeners since we really want to fully replace the original function.